.well-known/apple-app-site-association

```diff
@@ -63,6 +63,14 @@
                     {
                         "/": "/new/*",
                         "comment": "New Chat"
+                    },
+                    {
+                        "/": "/workspace/*",
+                        "comment": "Details of Workspace"
+                    },
+                    {
+                        "/": "/save-the-world/*",
+                        "comment": "Teachers Pages related"
```

Comment:
```suggestion
                        "comment": "Save the World"
```
I think this is fine, since we're going to be using these for different initiatives